samples/nrf5340/remote_shell/prj_uart.conf

```diff
@@ -7,8 +7,6 @@
 
 CONFIG_STDOUT_CONSOLE=y
 
-CONFIG_REMOTE_SHELL_UART_DEV="UART_0"
```

Comment:
An overlay with `ncs,remote-shell-uart = &uart0;` needs to be provided to cover this case.

Reply:
good catch, will fix it